src/lib/pipeline.ml

```diff
@@ -324,9 +328,11 @@ module Construct = struct
        make_target }
       bam_pair
 
-end
+  let rna_hla_typer fastq_sample = Rna_hla_typer fastq_sample
```

Comment:
The code is in `Pipeline` because of the non-extensibility we discussed yesterday (Vs circular dependencies).

What's important is how this kind of **user** code looks like:
https://github.com/hammerlab/biokepi/blob/master/src/lib/common_pipelines.ml#L54-L58

If there was an `.mli` file the GADT would be hidden (or at least `private`); and the `Pipeline.Construct` module is the one that should really be exposed as “API”; we want the user to use functions that say which tool/configuration is used: `mutect`, `mutect2`, `somaticsniper` and `seq2HLA` because the tools even when they have the same goals; they do extremely different things and have very different results.

If a user starts using `Construct.rna_hla_typer` it means nothing, and looks like the "default" HLA typer may change in the future. Biokepi authors should not choose which tool is best/default for a task, that's up to the user (who most likely knows more biology than us, and what they actually want).